### PR TITLE
[12.0] [MIG] sale_merge_draft_invoice

### PR DESCRIPTION
**Pull request by nikul-serpentcs**
Odoo can merge sales orders into a single invoice, grouped by partner and
currency when the sales orders are selected and invoiced together. This module
extends the grouping to also consider any pre-existing draft invoices and merge
them.

This feature is activated from Sales / Settings and assigned to the company
settings.

Users assigned to the group 'Change sale to invoice merge proposal'
have the possibility to activate or deactivate this option at the moment of
invoicing.

If you also need to group on fields other than partner and currency, the OCA
module sale_invoice_group_method allows you to specify alternative grouping
methods and is fully compatible with this module.

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please check CIs status

**Comment by nikul-serpentcs** [comment]
> Please check CIs status

Yes Sure @pedrobaeza

File: sale_merge_draft_invoice/README.rst
Comment (jimhoefnagels) [inline sale_merge_draft_invoice/README.rst:104]: Hi,
'~' causes error on install
docutils.utils.SystemMessage: <string>:105: (SEVERE/4) Title level inconsistent:

Authors
~~~~~~~
Kind regards
Jim

Comment (nikul-serpentcs) [reply sale_merge_draft_invoice/README.rst:104]: Yes, I know , That's way `Mr.Travis` is Red

File: sale_merge_draft_invoice/readme/USAGE.rst
Comment (rven) [inline sale_merge_draft_invoice/readme/USAGE.rst]:
```suggestion
* Create an Invoice from a single Sale Order
```

File: sale_merge_draft_invoice/readme/USAGE.rst
Comment (rven) [inline sale_merge_draft_invoice/readme/USAGE.rst]:
```suggestion
```

File: sale_merge_draft_invoice/readme/USAGE.rst
Comment (rven) [inline sale_merge_draft_invoice/readme/USAGE.rst]:
```suggestion
* Create Invoices from a group of Sales Orders
```

File: sale_merge_draft_invoice/readme/USAGE.rst
Comment (rven) [inline sale_merge_draft_invoice/readme/USAGE.rst]:
```suggestion
```

**Review by rven — CHANGES_REQUESTED** [review]
@nikul-serpentcs These changes solve the README.rst problem

**Comment by nikul-serpentcs** [comment]
@rven Thanks for help but still ..... :sob:

**Comment by rven** [comment]
It solves the problem, but the README file is not generated automtically at this point. Can you generate It also and try if It helps?

**Comment by rven** [comment]
The error is odoo.exceptions.UserError: ("You try to install module 'sale_merge_draft_invoice' that depends on module 'sale_order_action_invoice_create_hook'.\nBut the latter module is not available in your system.", '')

**Comment by pedrobaeza** [comment]
Rebase the branch over current 12.0 branch

**Comment by rven** [comment]
> Rebase the branch over current 12.0 branch

Doesn't help because the PR https://github.com/OCA/sale-workflow/pull/774 is not yet approved of the sale_order_action_invoice_create_hook

**Comment by pedrobaeza** [comment]
OK, then you can include temporarily in oca_dependencies.txt a reference to that branch or wait until the other PR is merged and then rebase.

**Comment by pedrobaeza** [comment]
Superseeded by #872

**Pull request closed without merging**
